src/renderer/react/components/form/password/PasswordStrength.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import zxcvbn from 'zxcvbn';
+import memoize from 'fast-memoize';
+
+import { concat } from 'services/css';
+
+import { scores, SCORES } from './password-strength.messages';
+import classNames from './password-strength.module.scss';
+
+export default class PasswordStrength extends React.PureComponent {
+  static propTypes = {
+    /** The user password. */
+    password: PropTypes.string,
+  };
+
+  static defaultProps = {
+    password: '',
+  };
+
+  /**
+   * (MEMOIZED).
+   * Calculates password strength through zxcvbn.
+   *
+   * @param {string} password - The user password.
+   * @returns {{score: number}} - User password's strength.
+   * @example const { score } = this.getStrength('azerty123');
+   */
+  getStrength = memoize(password => (password.length === 0 ? { score: -1 } : zxcvbn(password)));
```

Comment:
If you only use the score, maybe return it directly ?

Reply:
We may use other parts of the strength (hints, etc... when design is done.)